### ISSUE TITLE
improving efficiency for sentiment analysis in amazon dataset

### ISSUE THREAD

**Issue by psyuktha** [Recode-Hive/Scrape-ML#116]
I have noticed that the sentiment analysis on the amazon dataset [Scrape-ML](https://github.com/Recode-Hive/Scrape-ML/tree/main)/[amazon_scrapping](https://github.com/Recode-Hive/Scrape-ML/tree/main/amazon_scrapping)/sentiment_analysis_33.ipynb is just giving an accuracy of 83%.
It is designed on LogisticRegression model from sklearn

I would like to increase the accuracy by using other model architectures from Tensorflow hub 

By solving this users can have more reliable predictions from the model
 
I am a GSSOC'24 contributor. Please assign this issue to me.

**Comment by Laxmanpratapsingh** [comment]
I would like to work on this, please assign this to me

**Comment by sanjay-kv** [comment]
Let me know @psyuktha if you want to collaborate with @Laxmanpratapsingh

**Comment by sanjay-kv** [comment]
I will check this and update you

**Comment by Laxmanpratapsingh** [comment]
Sir, Please assign me any project at this level

**Comment by DiyaShah7** [comment]
Kindly assign me this issue

**Comment by sanjay-kv** [comment]
No, the idea is you use csv file from scrape data and analyse that.

**Comment by psyuktha** [comment]
ok I got it

**Comment by beingtony** [comment]
@sanjay-kv will you please assign this to me

**Comment by sanjay-kv** [comment]
@beingtony bro please stop spam with comments.

![image](https://github.com/Recode-Hive/Scrape-ML/assets/30715153/3fd9e810-254c-4daa-b6c2-e22281492f34)

I have mentioned multiple times, you can see in the above picture its already assigned. feel free to create new issue. and  I belive this is already completed.
